src/program.js

```diff
@@ -276,6 +276,11 @@ Example: $0 --help run.
       requiresArg: true,
       type: 'array',
     },
+    'no-input': {
+      alias: 'no-interactive',
```

Comment:
I don't think we need to add an alias. You can remove this. Even though `--no-input` isn't perfect, it's concise. The less aliases we have the better.